examples/metrics/observer_example.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+This example shows how the Observer metric instrument can be used to capture
+asynchronous metrics data.
+"""
+import psutil
+
+from opentelemetry import metrics
+from opentelemetry.sdk.metrics import LabelSet, Meter
+from opentelemetry.sdk.metrics.export import ConsoleMetricsExporter
+from opentelemetry.sdk.metrics.export.batcher import UngroupedBatcher
+from opentelemetry.sdk.metrics.export.controller import PushController
+
+# Configure a stateful batcher
+batcher = UngroupedBatcher(True)
+
+metrics.set_preferred_meter_implementation(lambda _: Meter(batcher))
+meter = metrics.meter()
+
+# Exporter to export metrics to the console
+exporter = ConsoleMetricsExporter()
+
+# Configure a push controller
+controller = PushController(meter, exporter, 2)
+
+
+# Callback to gather cpu usage
+def get_cpu_usage_callback(observer):
+    for (number, percent) in enumerate(psutil.cpu_percent(percpu=True)):
+        label_set = meter.get_label_set({"cpu_number": str(number)})
+        observer.observe(percent, label_set)
+
+
+meter.register_observer(
+    get_cpu_usage_callback,
+    "cpu_percent",
+    "per-cpu usage",
+    1,
```

Comment:
is 1 the period? for an example, might be good to call out the argument name explicitly.

Reply:
actually looking at the register_observe api, looks like the 4th argument is "unit". is 1 the correct value? I'm not sure what unit that corresponds to.

Reply:
It's the unit, 1 for dimensionless units. https://github.com/open-telemetry/opentelemetry-python/pull/341#discussion_r376758482.
I'll update all the metrics example to use explicit parameter names.

Reply:
Ah ok, it is possible to document somewhere that this value should follow the UCUM convention? Would be helpful for those new to the library.